sites/all/themes/dccr2015/templates/views/views-view-unformatted--sponsors.tpl.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * View that shows the sponsors grouped by Level
+ *
+ * @ingroup views_templates
+ */
+?>
+<?php if (!empty($title)): ?>
+  <h3 class="<?php print(strtolower($title)); ?>"><?php print $title; ?></h3>
+<?php endif; ?>
+
+<div class="sponsors-wrapper">
+	<?php foreach ($rows as $id => $row): ?>
+	  <div<?php if ($classes_array[$id]) { print ' class="' . $classes_array[$id] .'"';  } ?>>
```

Comment:
That's ok for me. I hadn't checked the base template. I just was wondering about the variable name; it sounds like an array.